src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -384,13 +516,24 @@ def post(self, request: Request, group: Group) -> Response:
             )
             profile = None
 
+        # get trace tree for this event
+        try:
+            trace_tree = self._get_trace_tree_for_event(event, group.project) if event else None
+        except Exception as e:
+            logger.exception(
+                "Failed to get trace tree for event",
+                extra={"group_id": group.id, "created_at": created_at, "exception": e},
```

Comment:
Don't actually need to include the exception in extra unless you really need to on a `logger.exception`, it automatically already includes the current exception & its stacktrace